components/automate-ui/src/app/page-components/create-user-modal/create-user-modal.component.ts

```diff
@@ -125,6 +125,13 @@ export class CreateUserModalComponent implements OnInit, OnDestroy {
     }
   }
 
+  handlePasswordInput(event: KeyboardEvent): void {
+    this.createUserForm.get('confirmPassword').updateValueAndValidity();
```

Comment:
I was wondering why this situation is unique, requiring bringing in this extra call. But I commented it out locally and all errors reacted just the same, so i could not find a need for it. Do you have a use case where this becomes significant?

(Same question for user-details.component -- commenting it out does not seem to change any behavior.)

Reply:
This specifically handles the case where a user enters a `password` and `confirmPassword`, then goes back and changes the original password. Since the match validator is only activated by changes to `confirmPassword`, we have to manually trigger a new check on confirmPassword here

Reply:
Ah, quite so! Thanks for pointing that out.
Could you add a brief in-code comment to that effect so others will know as well?